metadata.rb

```diff
@@ -4,10 +4,13 @@
 name             "drupal"
 description      "Installs/Configures drupal"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.0.3"
+version          "2.0.4"
 recipe           "drupal", "Installs and configures Drupal"
 recipe           "drupal::cron", "Sets up the default drupal cron"
 recipe           "drupal::drush", "Installs drush - a command line shell and scripting interface for Drupal"
+depends "mysql", "~> 5.0"
```

Comment:
So this looks funny because you'll be doing `depend` on `mysql` twice now (see line 15 which is a string array thingy within a foreach.